tests/std/tests/P0475R1_P0591R4_uses_allocator_construction/test.cpp

```diff
@@ -184,7 +185,8 @@ struct MoveOnlyType {
     MoveOnlyType(MoveOnlyType&&) = default;
 };
 
-void test_LWG3527() { // COMPILE-ONLY
+void test_lwg_3527() { // COMPILE-ONLY
+    // LWG-3527: "uses_allocator_construction_args handles rvalue pairs of rvalue references incorrectly"

```

Comment:
Yes, it's repetition, but it's useful. It allows users to find this piece of code using "LWG-xxxx". So I would like to keep this repetition. Although it may not be that important, let me know if you still want to remove this repetition.

https://github.com/microsoft/STL/search?q=LWG-2415&type=code

Reply:
I like the searchability rationale for this repetition. :+1:

Reply:
Looking at this some more, I think that we could achieve consistency the other way - mention GH-nnn in comments, because the GitHub Issues & Pull Requests extension for VSCode (if one is using that) will show cool hover cards for such citations.

Reply:
I'll push a change.